Makefile

```diff
@@ -160,6 +163,13 @@ $(PROMETHEUS): | $(BIN_DIR)
 	@echo "Downloading Prometheus"
 	curl -L "https://github.com/prometheus/prometheus/releases/download/v$(PROMETHEUS_VERSION)/prometheus-$(PROMETHEUS_VERSION).$$(go env GOOS)-$$(go env GOARCH).tar.gz" | tar --strip-components=1 -xzf - -C $(BIN_DIR)
 
+$(LOKI): | $(BIN_DIR)
+	@echo "Downloading Loki"
+	(cd $(BIN_DIR) && curl -O -L "https://github.com/grafana/loki/releases/download/v$(LOKI_VERSION)/loki-linux-$$(go env GOARCH).zip" && \
```

Comment:
we like to track our binary dependencies using the tools.go pattern. See the examples below in the Makefile and the tools.go at the root of the repo to add a new dependency

Reply:
this will also fix the CI test, which is complaining that the `unzip` binary isn't available

Reply:
Need to figure out the appropriate replace between loki/prometheus/azure-autorest first.

Reply:
grrr we always hit this. In thanos we use this: https://github.com/thanos-io/thanos/blob/master/go.mod#L74

Reply:
Unfortunately! let's keep it for now as it is and add unzip to the CI settings. WDYT?

Reply:
If you don't mind, I'll add it to your commit directly so we can have it here rather than mess with the CI just to merge this PR